src/subsplit_dag.cpp

```diff
@@ -314,7 +325,7 @@ void SubsplitDAG::IterateOverRootsplitIds(const std::function<void(size_t)> &f)
 
 RootedIndexerRepresentation SubsplitDAG::IndexerRepresentationOf(
     const Node::NodePtr &topology, size_t default_index) const {
-  return RootedSBNMaps::IndexerRepresentationOf(gpcsp_indexer_, topology,
+  return RootedSBNMaps::IndexerRepresentationOf(GetGPCSPIndexer(), topology,
```

Comment:
THIS could be a major bottleneck. Take a look around at where we use this... do we ever call it for lots of topologies? If so, we are rebuilding the indexer each time.